Dockerfile

```
@@ -1,4 +1,4 @@
-ARG PYTHON_VERSION=3.9
+ARG PYTHON_VERSION=3.11
 FROM python:${PYTHON_VERSION}-slim AS base
```

Not related to this PR, but it would be good to use [our own Docker image](https://github.com/Vandebron/docker-images/tree/main/python/3.11-slim-bookworm):
```suggestion
FROM public.ecr.aws/vdb-public/python:${PYTHON_VERSION}-slim-bookworm AS base
```